src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/MagicNumberCheck.java

```diff
@@ -68,6 +68,11 @@
  * Default value is {@code false}.
  * </li>
  * <li>
+ * Property {@code ignoreAnnotationElementDefaults} -
+ * Ignore magic numbers in annotation elements defaults.
+ * Default value is {@code false}.
```

Comment:
from issue comment - https://github.com/checkstyle/checkstyle/issues/4383#issuecomment-504678505

> This Check need to skip such cases by default to my mind

please explain ignoring of this request.

Reply:
This PR was opened before comment and I missed it.

I corrected code.